inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/node/AgentClusterNodeOperator.java

```diff
@@ -88,4 +163,95 @@ protected void setTargetEntity(ClusterNodeRequest request, InlongClusterNodeEnti
                     String.format("serialize extParams of Agent ClusterNode failure: %s", e.getMessage()));
         }
     }
+
+    public void updateModuleConfig(String ip, String clusterName) {
+        try {
+            ConfigRequest configRequest = new ConfigRequest();
+            LOGGER.info("begin to update module config for cluster name={}, ip={}", clusterName, ip);
+            if (StringUtils.isBlank(clusterName) || StringUtils.isBlank(ip)) {
+                LOGGER.info("no need to update module config when ip or cluster name is null");
+                return;
+            }
```

Comment:
blank check move to the top of the method

Reply:
Fixed.